submissions/maryOkosun/javascript/productList.js

```diff
@@ -0,0 +1,24 @@
+var productList = [
+	{
+		productName:"Tecno Spark",
+		imageUrl:"https://www.google.com/imgres?imgurl=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1511707171634-5f897ff02aa9%3Fixlib%3Drb-1.2.1%26ixid%3DeyJhcHBfaWQiOjEyMDd9%26w%3D1000%26q%3D80&imgrefurl=https%3A%2F%2Funsplash.com%2Fs%2Fphotos%2Fmobile-phone&tbnid=LVNFDi1BCzVPKM&vet=12ahUKEwiG0qSyi4LtAhUIEhoKHa3kDoAQMygAegUIARCgAQ..i&docid=iPam8aYHdwb-BM&w=1000&h=1000&q=phones%20images%20download&ved=2ahUKEwiG0qSyi4LtAhUIEhoKHa3kDoAQMygAegUIARCgAQ", 
+		productPrice:"$250"
+	},
+	{
+		productName:"Infinix",
+		imageUrl:"https://www.google.com/imgres?imgurl=https%3A%2F%2Fi.pcmag.com%2Fimagery%2Farticles%2F04kfhQ8iMwvR3fdyfbkQ1bj-3..1569486352.jpg&imgrefurl=https%3A%2F%2Fwww.pcmag.com%2Fnews%2Fgalaxy-s10-plus-beats-all-flagship-phones-for-download-speeds&tbnid=jH3G8aIy1MI9fM&vet=12ahUKEwiG0qSyi4LtAhUIEhoKHa3kDoAQMygKegUIARC0AQ..i&docid=OOvnvqLkSTg1iM&w=810&h=456&q=phones%20images%20download&ved=2ahUKEwiG0qSyi4LtAhUIEhoKHa3kDoAQMygKegUIARC0AQ", 
+		productPrice:"$400"
+	},
+	{
+		productName:"Tecno Cammon",
+		imageUrl:"https://www.google.com/imgres?imgurl=https%3A%2F%2Fimages.ctfassets.net%2Fd6skzop43my5%2F25VOo3VCXV8PpXXWtptIkK%2F0fa1442cbcae92076719ed9161b6574e%2Fhmd-home-phones-20191219.jpg%3Fq%3D50&imgrefurl=https%3A%2F%2Fwww.hmdglobal.com%2F&tbnid=r2XgcLf1fgqH6M&vet=12ahUKEwibopHhi4LtAhVN0RoKHS2TBgAQMygFegUIARCsAQ..i&docid=M-gE3rUyVsfVwM&w=1200&h=1200&q=phones%20images%20url&ved=2ahUKEwibopHhi4LtAhVN0RoKHS2TBgAQMygFegUIARCsAQ", 
+		productPrice:"$450"
+	}
+]
+for (var i = 0, l=productList.length; i< l; i++) {
+	console.log(productList[i].productName + " with " + productList[i].imageUrl + " is sold for " + productList[i].productPrice);
+}
+
```

Comment:
Kindly remove all this white spaces here

Reply:
I am sorry. I do not even know how the deletion happened. I have fixed the issues with the whitespace. Kindly review. Thanks